tests/ftests/systemd.py

```diff
@@ -27,3 +29,91 @@ def is_delegated(config, scope_name):
                 # until we figure out something better :(
                 return True
             raise re
+
+    # This function builds the absolute path of the cgroup, using the default
+    # systemd path and checks if the cgroup exists. The function can build
+    # path for all three cgroup setup modes, including empty cgroup v2.
+    @staticmethod
+    def cgroup_exists(config, ctrl_name, cgroup_name, ignore_systemd=False):
```

Comment:
Hmmm, thinking out loud here...  Currently you aren't using `config` but would we need it if this function were run within a container?  I think so

Reply:
Sure, that will make it more `bash` than `Python` :)

Reply:
I don't have a strong preference here.  If you haven't changed it, leave it as is.  This is internal to the tests and we can change its parameters later with no repercussions.

Reply:
Sure, I have re-written it :)